e2e/nfs.go

```diff
@@ -43,7 +43,7 @@ var (
 	nfsNodePluginPSP   = "csi-nodeplugin-psp.yaml"
 	nfsRookCephNFS     = "rook-nfs.yaml"
 	nfsDeploymentName  = "csi-nfsplugin-provisioner"
-	nfsDeamonSetName   = "csi-nfs-node"
+	nfsDeamonSetName   = "csi-nfsplugin"
```

Comment:
This can be placed in the same commit as where the name is changed, grouping logical changes together. As a rule, each commit should be fully testable, after each commit e2e should be able to run without failures. We don't have that as a strict requirement, but consider that in future PRs. (if there is breakage, `git bisect ..` can not easily be used anymore)